src/decision/test/lidar-decision-test.cpp

```diff
@@ -116,27 +120,29 @@ class LidarDecisionTest : public testing::Test {
 
 TEST_F(LidarDecisionTest, mergeSimilarObstaclesPreSortedTest){
 
-    LidarDecision::mergeSimilarObstacles(sorted_obstacles, 0.11);
+    LidarDecision::mergeSimilarObstacles(sorted_obstacles, 0.11, distance);
     EXPECT_EQ(4, sorted_obstacles.size());
 }
 
 TEST_F(LidarDecisionTest, mergeSimilarObstaclesUnSortedTest){
     // Create some test obstacles in random order
     // (in terms of angle)
-    LidarDecision::mergeSimilarObstacles(unsorted_obstacles, 0.11);
+    LidarDecision::mergeSimilarObstacles(unsorted_obstacles, 0.11, distance);
     EXPECT_EQ(4, unsorted_obstacles.size());
 }
 
 TEST_F(LidarDecisionTest, findObstaclesTest){
-    std::vector<LidarObstacle> found_obstacles = LidarDecision::findObstacles(scan1, 0.1);
-    EXPECT_EQ(4, found_obstacles.size());
+    std::vector<LidarObstacle> found_obstacles = LidarDecision::findObstacles(scan1, 0.1, distance);
+    EXPECT_EQ(6, found_obstacles.size());
     // Check that all obstacles are at acceptable ranges
     for (LidarObstacle obstacle : found_obstacles){
         EXPECT_GE(obstacle.getAvgDistance(), 20);
         EXPECT_LE(obstacle.getAvgDistance(), 40);
     }
 }
 
+
+
 int main(int argc, char **argv) {
     testing::InitGoogleTest(&argc, argv);
     ::testing::InitGoogleTest(&argc, argv);
```

Comment:
Note that you will like likely have to create new fake scans for these tests. You can declaring and fill them in the same way that `scan1` is declared and filled.